kaos/api/validators.py

```diff
@@ -0,0 +1,49 @@
+"""This module defines the validation decorators used by the various view functions."""
+
+from functools import wraps
+
```

Comment:
why the new entries?

Reply:
What entries?

Reply:
Oh, the empty line between each import..

Reply:
That's the coding style:

All python imports
// New Line
Library Imports
// New Line
// Current Project Imports

At least that's what I follow